src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius/sylius_mailer.yml

```diff
@@ -19,6 +19,9 @@ sylius_mailer:
         shipment_confirmation:
             subject: sylius.emails.shipment_confirmation.subject
             template: "@SyliusCore/Email/shipmentConfirmation.html.twig"
+        shipment_confirmation_resent:
```

Comment:
Is separate email necessary business wise or it's only due BC promise, which disallows to move `AdminBundle\EmailManager\ShipmentEmailManager` to the core?

What about copying original `ShipmentEmailManager` as is, so there will be both
```php
AdminBundle\EmailManager\ShipmentEmailManager->sendConfirmationEmail
CoreBundle\EmailManager\ShipmentEmailManager->sendConfirmationEmail
```
but adding deprecation to the old one? So no new email is necessary.